authzed/api/v1/experimental_service.proto

```diff
@@ -29,6 +30,17 @@ service ExperimentalService {
         body: "*"
       };
     }
+
+  // BulkExportRelationships is the fastest path available to exporting
+  // relationships from the server. It is resumable, and will return results
+  // in an order determined by the server.
+  rpc BulkExportRelationships(BulkExportRelationshipsRequest)
```

Comment:
Only nitpick here is that there's no duality with the other bulk command. It should either be Load/Save or Import/Export

Reply:
Agreed